app/controllers/items_controller.rb

```diff
@@ -373,15 +437,18 @@ def self.normalize_source_id(new_source_id)
   def source_id
     new_id = ItemsController.normalize_source_id(params[:new_id])
     @object.set_source_id(new_id)
-    # TODO: the content= and content_will_change! calls belong in dor-services, for this method and other similar methods.
-    # can then clean up allow(idmd).to receive(:"ng_xml") (and "content_will_change!") in tests.  getting rid of "Metadata\.content\s*=\s*"
+    # TODO: the content= and content_will_change! calls belong in dor-services,
+    # for this method and other similar methods. can then clean up
+    # allow(idmd).to receive(:"ng_xml") (and "content_will_change!") in tests.
+    # getting rid of "Metadata\.content\s*=\s*"
```

Comment:
a personal gripe that i've made plenty of times already:  i would love to turn off the line-length complaint in hound and rubocop.  i don't find this re-written paragraph more readable, and i think most of the times that hound (or whatever) encourages short lines, we'd be better off ignoring the suggestion.  there are times when a line should be shortened.  if it's that egregious, someone can call it out during code review.